updatezinfo.py

```diff
@@ -17,9 +19,19 @@ def main():
 
     if not os.path.isfile(metadata['tzdata_file']):
         print("Downloading tz file from iana")
-        request.urlretrieve(os.path.join(metadata['releases_url'],
-                                         metadata['tzdata_file']),
-                            metadata['tzdata_file'])
+        for x in range(1, 6):
```

Comment:
This is not necessary. We can restart the CI builds if they fail, and anyone actually using the file can just re-run the script of the download fails.